PhysicsTools/NanoAOD/python/taus_cff.py

```diff
@@ -63,6 +147,7 @@ def _tauId6WPMask(pattern,doc):
        idAntiEle = _tauId5WPMask("againstElectron%sMVA6", doc= "Anti-electron MVA discriminator V6"),                         
        idMVAnewDM = _tauId6WPMask( "by%sIsolationMVArun2v1DBnewDMwLT", doc="IsolationMVArun2v1DBnewDMwLT ID working point"),
        idMVAoldDM = _tauId6WPMask( "by%sIsolationMVArun2v1DBoldDMwLT", doc="IsolationMVArun2v1DBoldDMwLT ID working point"),
+       idMVAoldDMnew = _tauId7WPMask( "by%sIsolationMVArun2v1DBoldDMwLTNew", doc="IsolationMVArun2v1DBoldDMwLT ID working point (New)"),
```

Comment:
could 'new' become '2017' or '94X' or anything more specific?  ("new" is a relative concept)

Reply:
I agree and will correct "new"-> "2017v1". Of course at some point it will be updated to something different (2017 will become default without any suffix and other "new" will appear), but hopefully users will deal with it.